MediaRecorder.bs

```diff
@@ -580,6 +580,7 @@ dictionary MediaRecorderOptions {
   unsigned long audioBitsPerSecond;
   unsigned long videoBitsPerSecond;
   unsigned long bitsPerSecond;
+  boolean audioConstantBitRate;
```

Comment:
nit: maybe call it BitsPerSecond to match the other members

Reply:
`BitsPerSecond` is a bit verbose, especially in the case of `audioConstantBitsPerSecond`, but you are right that it would be better to be consistent. Will update.